packages/block-editor/src/components/inserter/preview-panel.js

```diff
@@ -16,31 +15,24 @@ import BlockCard from '../block-card';
 import BlockPreview from '../block-preview';
 
 function InserterPreviewPanel( { item } ) {
-	const { name, title, icon, description, initialAttributes } = item;
-	const hoveredItemBlockType = getBlockType( name );
+	const { name, title, icon, description, initialAttributes, example } = item;
 	const isReusable = isReusableBlock( item );
 	return (
 		<div className="block-editor-inserter__preview-container">
 			<div className="block-editor-inserter__preview">
-				{ isReusable || hoveredItemBlockType?.example ? (
```

Comment:
I think we need this in case `example` is undefined, which is the case for block style previews.
Issue: https://github.com/WordPress/gutenberg/issues/43836
PR https://github.com/WordPress/gutenberg/pull/43883

Reply:
Thanks for spotting that.

I thought this component was only part of the Inserter, so didn't think to test that. Perhaps it should be renamed.